package.json

```diff
@@ -16,19 +16,23 @@
     "@emotion/react": "^11.10.5",
     "@emotion/styled": "^11.10.5",
     "@mui/material": "^5.11.7",
+    "@reduxjs/toolkit": "^1.9.1",
+    "@vanilla-extract/css": "^1.9.3",
+    "classnames": "^2.3.2",
```

Comment:
you should add also @types/classnames there are eslint errors because of lack of types

Reply:
I don't see any errors in my code.
Also @types/classnames is deprecated.
![obraz](https://user-images.githubusercontent.com/30327956/217340128-40110849-91f3-4144-a8e7-b7bbff60ffd6.png)